### ISSUE TITLE
Is this does not work or what?

### ISSUE THREAD

**Issue by odil24** [Patoche692/Nexus#11]
Uploading nexus.mp4…

Any specific needed? why is it black screen?

**Comment by Patoche692** [comment]
Nothing specific is needed, it should work on your side. It seems like the program is not even attempting to read the file. Are you sure to have read permission in your folder? Maybe try moving the scene file in the same folder as the executable.

**Comment by odil24** [comment]
OBJLoader: Error loading model D:\Nexus Render engine\assets\canelle_et_fromage\canelle_et_fromage.glb

**Comment by Patoche692** [comment]
Have you tried with other simpler scenes like cornell box? If it does not work maybe running the program as administrator would get you past the read permissions.

**Comment by odil24** [comment]
OBJLoader: Error loading model D:\Nexus Render engine\assets\cornell_box\cornell_box.glb

**Comment by odil24** [comment]
Any dependencies maybe am I must install?

**Comment by Patoche692** [comment]
I don't think it is missing some dependencies. Did you try running the program as administrator? Otherwise there is something wrong in my code.

**Comment by odil24** [comment]
> I don't think it is missing some dependencies. Did you try running the program as administrator? Otherwise there is something wrong in my code.

Yes, it is. I runned with and without as admin

**Comment by odil24** [comment]
Any news?

**Comment by hattmt** [comment]
I have tested the realease and it worked, I have also built the project and it's working as well; maybe you have not followed all  the 
 steps that are discribed in the Readme ?

Be careful : The scene is not included in the release . You need to open one and find it in the viewport.
